tests/unit/modules/test_github.py

```diff
@@ -1,13 +1,224 @@
+from unittest import mock
+from unittest import TestCase
+
 import pytest
+import salt.modules.config as config_module
 import salt.modules.test as testmod
 import saltext.github.modules.github as github_module
+from salt.exceptions import CommandExecutionError
 
 
 @pytest.fixture
 def configure_loader_modules():
     module_globals = {
-        "__salt__": {"test.echo": testmod.echo},
+        "__salt__": {"test.echo": testmod.echo, "config.option": config_module.option},
+        "__opts__": {"mock": {"ruleset_type": "mocked_value"}},
     }
     return {
         github_module: module_globals,
+        config_module: module_globals,
+    }
+
+
+@pytest.fixture
+def params():
+    return {
+        "ruleset_type": "repo",
+        "owner": "mock_owner",
+        "repo_name": "mock_repo_name",
+        "org_name": "mock_org_name",
+        "header_dict": {"Authorization": "mock_token"},
+    }
+
+
+def test__check_params():
+    ## test no valid config
+    with TestCase().assertRaises(CommandExecutionError):
```

Comment:
**chore (blocking):** I think this is old syntax. you should just be able to expect a raise like this:

```suggestion
    with pytest.raises(CommandExecutionError):
```